packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts

```diff
@@ -706,21 +706,6 @@ describe('NegotiatorHelpers', () => {
           schema: { type: 'string' },
         });
       });
-
-      it('and cannot find example and dynamic does not exist throw error', () => {
```

Comment:
I see you replaced the logic, with returning `{ code, mediaType }`.
Why remove this test VS update it to match what you return?

Reply:
Restored.